apps/demo-app/package.json

```
@@ -4,8 +4,8 @@
   "version": "1.0.0-beta.63",
   "type": "module",
   "scripts": {
-    "dev": "turbo run vite --filter demo-app",
-    "vite": "vite",
+    "dev": "turbo run turbo:dev --filter demo-app",
```

IMHO, we should use the `watch` command, which will rebuild dependencies if necessary:
```suggestion
    "dev": "turbo watch turbo:dev --filter demo-app",
```
I got it to work now, maybe we can sync if the issue only affected me.

---

Watch does not work as expected for me. E.g. if using `turbo watch` in the demo app and then e.g. make changes to the OnyxButton, turbo will re-build `sit-onyx` like it should but the demo app will be shut down and not work any longer...

---

-_- `turbo watch` broke again, so I can't even test it